src/gui/selectivesyncwidget.cpp

```diff
@@ -174,7 +175,7 @@ void SelectiveSyncWidget::recursiveInsert(QTreeWidgetItem *parent, QStringList p
         }
 
         pathTrail.removeFirst();
-        recursiveInsert(item, pathTrail, path, size);
+        recursiveInsert(item, pathTrail, path, size, showChildIndicator);
```

Comment:
Could you add a comment how showChildIndicator works in this recursion?
As it isn't changed, I'd expect it to define the indicator for the whole sub tree.